### DISCUSSION TITLE
Villager radiant marriage/births query

### DISCUSSION THREAD

**Discussion by MissAlexandrina** [Luke100000/minecraft-comes-alive#772] [Q&A]
I'm wondering if I'm not fully understanding how the mechanics of villagers marrying one another/having children actually function. (I fully understand how the matchmaker rings work, as well as how the _player_ gets married).

Early on in my save, I had two villagers marry pretty much immediately after I arrived at the village (they weren't married when I first arrived, but they were by the end of the second in-game day) but since then, nothing seems to be happening. I spent about 90 in-game days in that same village, on-and-off, without the married villagers ever producing children, before one of the villagers died. I've since brought other villagers to the same village, and have spent now over 100 in-game days idling in the village waiting for something to happen. There are two villagers, one male and one female, who are single and do not have conflicting traits (neither are homosexual or bisexual, confirmed with the villager editor). Their home is set to the same house, I have 100+ hearts with both, and there is plenty of room in the village for expansion (total population is currently five with beds for ten villagers).

I've experimented with changing the config files to 0.5, 1 and 50 (my understanding is that with the default value, you'd expect a marriage to occur on average every twenty minutes; bumping the value up to one should force an update every minute) and I've also tried deleting and reinstalling the mod in case I somehow messed up the config file. The adventurers coming to the inn work as intended (I've had a few) but nothing else seems to be happening. (I've tried sneaking with the editor to see villagers' inventories too, as I know in the original version babies were held as items until they were ready to grow up).

In addition I've tried creating a debug save with two villagers on a superflat world, with the village fully registered using the blueprint, and idled for another fifteen days without anything happening.

(I do have a few other mods installed, but I _did_ have one automatic marriage taking place early on in the save and haven't added new mods since, so it shouldn't be a case of compatibility).

Is there something I need to do on my end in-game for these events to happen, or should they just take place at random?

**Comment by Luke100000** [comment]
You are overthinking this, they just marry until they reach a threshold, 50% of all adult villagers by default and changeable in the blueprint.
